kobo/apps/accounts/mfa/urls.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 from django.urls import include, path
 
-from .views import MfaListUserMethodsView, MfaLoginView, MfaTokenView
+from .views import MfaListUserMethodsView, MfaLoginView, MfaTokenView, MfaMethodActivationView
```

Comment:
Please (black) reformat, we are trying to respect the 80 characters limit (with up to 90 if variable names are way too long).

```python 
from .views import (
    MfaMethodActivationView,
    MfaListUserMethodsView,
    MfaLoginView,
    MfaTokenView,
)
```

Reply:
Ok, thanks, I'll set up the linters to avoid these,